### DISCUSSION TITLE
About License

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#3584] [Q&A]
Is the license in the form of a document after I make the payment?
How should I register and use the file after receiving it? Is there a detailed explanation? Is it possible to directly overwrite the SDK to remove the running logo and release Linux programs?
thanks.
![1732697215873](https://github.com/user-attachments/assets/48b2f237-92c8-4560-97d5-7dfd0a480f96)

@ajwfrost

**Comment by ajwfrost** [comment]
Yes essentially it's a file with an ID and a license key. Once you drop this into the right location on your machine (`~/.airsdk/adt.lic`) it would allow your Linux SDK to start working properly and removing the splash screen etc.

thanks

**Reply by Jason0247** [reply]
If I want to set the program developed by Air to boot on Linux system, do I need to purchase additional certificates?

**Reply by ajwfrost** [reply]
If it's an application that's intended to become the user interface of the computer itself then likely yes, there's a redistribution agreement that we have with different terms from Adobe. I think the main consideration though is about the installation of it: if you provide an application that the end user installs, then you can use the AIR SDK license agreement; if it's intended to be distributed already integrated together with a device then we have the separate "Integrated Product" license agreement...

So for example, we have a number of Automotive vendors who use AIR as the mechanism to display the user interface for infotainments systems on Windows/Linux/QNX, and those require separate licenses. Plus some devices that have touch-enabled displays such as smart ovens, supermarket scales, printers, gaming consoles etc - those would be considered for the other license agreement which is a royalty-based one (we have to report numbers to Adobe and pay them based on the number of units shipped).

If you check the [AIR SDK license agreement](https://airsdk.harman.com/assets/pdfs/HARMAN%20AIR%20SDK%20License%20Agreement.pdf) and don't think it would cover you, please contact me directly.

thanks

**Reply by Jason0247** [reply]
It was my expression error. In fact, what I want to implement is a small program similar to a video player. After AIR is generated, it is copied to the computer and can be set by the user to boot up.
I see that one way to boot Linux is through a desktop file. I'm not sure if AIR can automatically generate this file or if there are other ways to add it to the boot process.

**Reply by ajwfrost** [reply]
Ah - if you mean, you want your AIR application to boot up automatically when the user logs in to Linux, then this should be supported:
https://airsdk.dev/reference/actionscript/3.0/flash/desktop/NativeApplication.html#startAtLogin

thanks

**Reply by itlancer** [reply]
@ajwfrost 
When I check `NativeApplication::startAtLogin` last time it didn't work https://github.com/airsdk/Adobe-Runtime-Support/issues/861. Is it was fixed?

**Reply by ajwfrost** [reply]
Ah - okay so looking at the comments there, it requires the application to be "properly" installed on a platform - looks like it was dependent upon the certificate/verification process. Which should now be implemented properly, at least on macOS and Windows. We will have to check this on Linux (and will double-check the other OS too..)

thanks
